src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixAbstractMethodsStep.cs

```diff
@@ -19,43 +19,80 @@ namespace MonoDroid.Tuner
 	/// <summary>
 	/// NOTE: this step is subclassed so it can be called directly from Xamarin.Android.Build.Tasks
 	/// </summary>
-	public class FixAbstractMethodsStep : BaseStep
+	public class FixAbstractMethodsStep
+#if NET5_LINKER
```

Comment:
I'm wondering if it'd be better to split this to netcore version and keep the old one untouched

Reply:
I did this originally but changed it in response to https://github.com/xamarin/xamarin-android/pull/5748#discussion_r599262551.